bitnami/common/templates/_affinities.tpl

```diff
@@ -0,0 +1,94 @@
+{{/* vim: set filetype=mustache: */}}
+
+{{/*
+Return a soft nodeAffinity definition 
+{{ include "common.affinities.node.soft" (dict "key" "FOO" "values" (list "BAR" "BAZ")) -}}
+*/}}
+{{- define "common.affinities.node.soft" -}}
+preferredDuringSchedulingIgnoredDuringExecution:
+  - preference:
+      matchExpressions:
+        key: {{ .key }}
+        operator: In
+        values:
+          {{- range .values }}
+          - {{ . }}
+          {{- end }}
+    weight: 1
+{{- end -}}
+
+{{/*
+Return a hard nodeAffinity definition
+{{ include "common.affinities.node.soft" (dict "key" "FOO" "values" (list "BAR" "BAZ")) -}}
+*/}}
+{{- define "common.affinities.node.hard" -}}
+requiredDuringSchedulingIgnoredDuringExecution:
+  nodeSelectorTerms:
+    - matchExpressions:
+        key: {{ .key }}
+        operator: In
+        values:
+          {{- range .values }}
+          - {{ . }}
+          {{- end }}
+{{- end -}}
+
+{{/*
+Return a nodeAffinity definition
+{{ include "common.affinities.node" (dict "type" "soft" "key" "FOO" "values" (list "BAR" "BAZ")) -}}
+*/}}
+{{- define "common.affinities.node" -}}
+  {{- if eq .type "soft" }}
+    {{- include "common.affinities.node.soft" . -}}
+  {{- else if eq .type "hard" }}
+    {{- include "common.affinities.node.hard" . -}}
+  {{- end -}}
+{{- end -}}
+
+{{/*
+Return a soft podAffinity/podAntiAffinity definition
+{{ include "common.affinities.pod.soft" (dict "component" "FOO" "context" $) -}}
+*/}}
+{{- define "common.affinities.pod.soft" -}}
```

Comment:
Sorry, I have an OCD here... XD, I have another proposal for the name, could we put `pods` and `nodes` also XD, it would sound like

`common.affinities.pods.soft`

In the common templates, we have `affinities` for `pods` and they could be `soft` and `hard`. 

Also

`common.affinities.pods`

What do you think about it? do you agree?